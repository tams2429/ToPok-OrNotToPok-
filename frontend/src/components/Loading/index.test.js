import { render, fireEvent } from '@testing-library/react'
import { Loading } from './index' 

describe('Loading Component', () => {

  it('should render properly and match snapshot', () => {
    const loadingComponent = render(<Loading />)
    expect(loadingComponent.container).toMatchSnapshot()
  })
})