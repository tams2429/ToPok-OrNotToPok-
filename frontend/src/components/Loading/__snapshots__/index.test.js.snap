// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component should render properly and match snapshot 1`] = `
<div>
  <section
    class="loading-wrapper"
  >
    <p
      class="loading-text"
      data-testid="loading-text"
    />
    <div
      class="pokeball"
    />
  </section>
</div>
`;
